### PR TITLE
[W5.7][W14-2] Leong Eng Sea

### PR DESCRIPTION
**Pull request by ghost**
src/seedu/addressbook/Main.java: List all persons automatically after an add or delete command.

### CODE REVIEW

**Comment by YuPeiHenry** [comment]
Hey Eng Sea, good attempt at enhancing the add/delete command. I have some suggestions:
1. Structure the enhancement in a more systematic way, don't just introduce it using bootstrap code inside main.java. Code might become highly unsustainable after a few iterations of this.
2. Update your test cases, both JUnit and I/O. When Travis fails, I think it indicates that at least one of these tests have failed. Commits with failing tests should be fixed quickly to reduce the cost of fixing them later.
3. Using the Parser as such to simulate commands should be avoided. The Parser's main role is to parse actual input from the user, keeping its Single Responsibility.
Please consider my suggestions, it is really beneficial for you and our team.

File: src/seedu/addressbook/Main.java [hunk 1]
```diff
@@ -88,8 +88,16 @@ private void runCommandLoopUntilExitCommand() {
             recordResult(result);
             ui.showResultToUser(result);
 
+            if(userCommandText.contains("add") || userCommandText.contains("delete")) {
```
Comment (kexiaowen) [inline src/seedu/addressbook/Main.java:91]: Add a space after `if`

File: src/seedu/addressbook/Main.java [hunk 2]
```diff
@@ -88,8 +88,16 @@ private void runCommandLoopUntilExitCommand() {
...
+                command = new Parser().parseCommand("list");
+                CommandResult result1 = executeCommand(command);
+                recordResult(result1);
+                ui.showResultToUser(result1);
+
+            }
         } while (!ExitCommand.isExit(command));
-    }
+
+    }// while (!ExitCommand.isExit(command));
```
Comment (kexiaowen) [inline src/seedu/addressbook/Main.java:100]: Remove code that is commented out

File: src/seedu/addressbook/Main.java [hunk 3]
```diff
@@ -88,8 +88,16 @@ private void runCommandLoopUntilExitCommand() {
             recordResult(result);
             ui.showResultToUser(result);
 
+            if(userCommandText.contains("add") || userCommandText.contains("delete")) {
+                command = new Parser().parseCommand("list");
+                CommandResult result1 = executeCommand(command);
```
Comment (kexiaowen) [inline src/seedu/addressbook/Main.java:93]: Not a good idea to name your variables like result1, result2 etc. You can try to give a more informative name

**Review by kexiaowen — APPROVED** [review]
@leongengsea Hi Eng Sea, you might want to spend some more time looking into different components of the addressbook and try to do the enhancements from there. Henry has provided some thoughtful comments. You can take a look. Pls close this PR after reading through the comments.

**Pull request closed without merging**